app/src/main/java/org/mozilla/fenix/Experiments.kt

```diff
@@ -8,13 +8,17 @@ import android.content.Context
 import mozilla.components.service.fretboard.ExperimentDescriptor
 
 const val EXPERIMENTS_JSON_FILENAME = "experiments.json"
-const val EXPERIMENTS_BASE_URL = "https://settings.prod.mozaws.net/v1"
+// const val EXPERIMENTS_BASE_URL = "https://settings.prod.mozaws.net/v1"
+const val EXPERIMENTS_BASE_URL = "https://kinto.dev.mozaws.net/v1"
 const val EXPERIMENTS_BUCKET_NAME = "main"
-// TODO Change this after fenix-experiments is created
-const val EXPERIMENTS_COLLECTION_NAME = "focus-experiments"
+const val EXPERIMENTS_COLLECTION_NAME = "firefox-preview-features"
```

Comment:
hopefully we can rename this from focus to proper `firefox-preview`

Reply:
You must connect using VPN and create the buckets at https://settings-writer.prod.mozaws.net/v1/admin/